Solving Issue #1208 (#1246)
MicrosoftDocs/OfficeDocs-Exchange: Exchange/ExchangeOnline/mail-flow-best-practices/non-delivery-reports-in-exchange-online/fix-error-code-550-5-7-1-in-exchange-online.md

If the connectors are configured incorrectly, your Exchange administrator needs to rerun the Hybrid Configuration Wizard in the on-premises Exchange organization.

> Now, if you disabled an Active Directory account On-premises, the error that you will get this message:

Reviewer: ```suggestion
If you have disabled an Active Directory account On-premises, you will get the following error message:
```

```
Your message couldn't be delivered to the recipient because you don't have permission to send to it.
Ask the recipient's email admin to add you to the accept list for the recipient.
For more information, see DSN 5.7.129 Errors in Exchange Online and Office 365.
````

> - In case you want to cease all the communication with the mailbox on Exchange-Online the solution would be to delete the user completely On-Premises, instead of disabling it.

Reviewer: ```suggestion
- If you want to cease all communication with the mailbox on Exchange-Online, you need to delete the user completely on-premises, instead of disabling the account.
```

> - Another solution would be to remove his license but in this case, to stop receiving emails you'd need to create a transport rule, if you don't do this the user will be able to receive emails for about 30 days more after removal of the license

Reviewer: ```suggestion
- Another solution would be to remove the license, but in this case you'd need to create a transport rule to stop the user from receiving emails. If you don't do this, the user will still be able to receive emails for about 30 days after removal of the license.
```

> Please consider this scenario as part of the disabling workflow for a user on Exchange Online.

Reviewer: ```suggestion
Please consider this scenario as part of disabling workflow for a user on Exchange Online.
```
